repo2docker/buildpacks/conda/environment.yml

```diff
@@ -4,9 +4,10 @@ dependencies:
   - python=3.10
   - nodejs=18
   - pip
-  - ipywidgets==8.0.2
-  - jupyter-offlinenotebook==0.2.2
-  - jupyter-resource-usage==0.7.0
-  - jupyterlab==3.4.8
-  - jupyterhub-singleuser==3.1.1
-  - notebook==6.4.12
+  - ipywidgets==8.*   # https://github.com/jupyter-widgets/ipywidgets
+  - jupyter-offlinenotebook==0.2.*   # https://github.com/manics/jupyter-offlinenotebook
+  - jupyter-resource-usage==0.7.*   # https://github.com/jupyter-server/jupyter-resource-usage
+  - jupyter_server==1.*   # https://github.com/jupyter-server/jupyter_server
+  - jupyterhub-singleuser==3.*   # https://github.com/jupyterhub/jupyterhub
+  - jupyterlab==3.*   # https://github.com/jupyterlab/jupyterlab
+  - notebook==6.*   # https://github.com/jupyter/notebook
```

Comment:
These are teh changes to review in this PR. We go from various minor/patch versions to the latest available in the same major version.

jupyter-server was pinned explicitly, as it was indirectly pinned by jupyterlab 3.4 which when upgraded to jupyterlab 3.5+ wasn't indirectly pinned any more, and that broke tests we had and would been a significant change we want to do intentionally in a separate PR.

Reply:
R test failures shows up in the main branch as well, and tracking that is done now in https://github.com/jupyterhub/repo2docker/issues/1284. I believe them to be entirely unrelated from this PRs changes.